packages/turf-difference/test/out/create-hole.geojson

```diff
@@ -7,50 +7,54 @@
 				"fill-opacity": 1
 			},
 			"geometry": {
-				"type": "Polygon",
+				"type": "MultiPolygon",
```

Comment:
@rowanwins 🤔 ... you sure this is correct?... That was the initial issue was trying to handle the holes.

Reply:
Seems like the Inner blue should be a lighter shade.

![image](https://user-images.githubusercontent.com/550895/35068324-b941fd78-fba4-11e7-9752-2fa7240e0770.png)

Reply:
Correct - so martinez currently has an issue around this, I had a thought on how to resolve it last night lying in bed so I'll try and tackle it today

Reply:
FYI I have a [pull request in](https://github.com/w8r/martinez/pull/55) to resolve this issue upstream in martinez